src/views/datalist/data-list-directive.js

```diff
@@ -227,6 +279,80 @@ angular.module('patternfly.views').directive('pfDataList', function (pfUtils) {
             'Illegal use of pfDataList directive! ' +
             'Cannot allow both select box and click selection in the same data list.');
           }
+
+          if (!$scope.listId) {
+            $scope.listId = 'pf-data-list';
```

Comment:
If there are more than one on a page the app would need to pass in the ID via the listId parameter.